code/style.css

```diff
@@ -1,4 +1,288 @@
+  display: none;
+  font-size: 25px;
+}
+
+.bars {
+  display: none;
+}
+
+.big-news {
+  margin: 10px 10px 0 10px;
+  display: flex;
+  flex-direction: column;
+  background-color: whitesmoke;
+  position: relative;
+  align-items: center;
+}
+
+.gotland-image {
+  width: 100%;
+  height: auto;
+}
+
+.big-news-text:hover {
+  background-color:white;
+  text-shadow: 1px 1px grey;
+}
+
+h2 {
+  font-size: 28px;
+  text-transform: uppercase;
+  margin: 10px;
+}
+
+.small-text-big-news {
+  margin-top: 0px;
+  font-size: 20px;
+  margin: 0px 10px;
+}
+
+h3 {
+  font-size: 24px;
+  text-transform: uppercase;
+  margin: 10px 5px 5px 5px;
+}
+
+p {
+  margin: 0px 5px 25px 5px;
+}
+
+p:hover {
+  text-shadow: 1px 1px grey;
+}
+
+.more-news {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 10px;
+}
+
+.news-card {
+  background-color: whitesmoke;
+  width: 100%;
+  padding-top: 5px;
+  margin-bottom: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+img {
+  width: 80%;
+}
+
+.news-card:hover {
+  background-color: white;
+  border-top: 2px solid coral;
+}
+
+footer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  background-color: whitesmoke;
+  padding: 30px;
+  animation-name: disco-footer;
+  animation-duration: 3s;
+  animation-iteration-count: infinite;
+}
+
+.foot-text1, .foot-text2 {
+  padding-top: 25px;
+}
+
+.button {
+  padding: 8px 16px;
+  border-radius: 25px;
+  border: none;
+  background-color: coral;
+  color: whitesmoke;
+  font-size: 15px;
+}
+
+a {
+  text-decoration: none;
+  color: rgb(78, 77, 77);
+}
+
+/*tablet version*/
+@media (min-width: 668px) and (max-width: 1023px) {
+  h1 {
+    font-size: 50px;
+    left: 150px;
+  }
+
+    .fa-bars {
+    display: inline;
+  }
+
+  .bars {
+    display: flex;
+    align-items: center;
+  }
+
+    .big-news {
+      flex-direction: row;
+      justify-content: space-evenly;
+      align-items: center;
+      width: 100%;
+    }
+
+    .gotland-image {
+      width: 50%;
+    }
+    
+    .big-news:hover {
+      text-shadow: 1px 1px grey;
+    }
+
+    .big-news-text:hover {
+      background-color: whitesmoke;
+    }
+
+    .more-news {
+      flex-direction: row;
+      flex-wrap: wrap;
+      justify-content: space-evenly;
+      width: 100%;
+      padding-left: 2px;
+      padding-right: 2px;
+    }
+
+    .news-card {
+      width: 48%;
+    }
+
+    img {
+      width: 80%;
+    }
+
+    .button {
+      position: absolute;
+      left: 40%;
+    }
+
+    .foot-text1, .foot-text2 {
+      display: none;
+    } 
+}
+
+/*desktop version*/
+@media (min-width: 1024px) {
+  h1 {
+    font-size: 50px;
+    left: 150px;
+  }
+
+  .fa-bars {
+    display: none;
+  }
+
+  .nav-text1, .nav-text2 {
+    display: inline;
+  }
+
+  .big-news {
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: center;
```

Comment:
For `flex-direction: row;` I was told that we can use `'justify-items/content'` and `flex-direction: column;` uses `'align-items/content'`. I see in this code you are using both justify & align elements on `flex-direction: row;`. I have tried checking out the option `align-items: center;` and I did not see any changes on inspect screen. Is there a reason why you opted out both line 242 and 243 for .big news?